opba-protocols/xs2a-protocol-tests/xs2a-bdd-wiremock/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/wiremock/mocks/WiremockAccountInformationRequest.java

```diff
@@ -175,8 +176,8 @@ public SELF user_anton_brueckner_provided_initial_parameters_to_list_accounts_wi
 
         ExtractableResponse<Response> response = RestAssured
                  .given()
+                     .header(X_REQUEST_ID, DEFAULT_X_REQUEST_ID)
```

Comment:
Consent API calls should use generated request IDs (like it was before)

Reply:
random UUID is generated